cypress/integration/upload-download.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { UploadPage } from "../page/index.ts";
+
+let uploadPage: UploadPage;
+
+describe("Upload file", () => {
+  before(() => {
+    uploadPage = new UploadPage();
+    Cypress.on("uncaught:exception", () => false);
+    uploadPage.visitUploadPage();
+  });
+
+  it("Upload file", () => {
+    uploadPage.uploadFile();
+    uploadPage.getTitleAfterUpload();
```

Comment:
I think the assertion should be clear inside the test file instead of it being done inside the POM, but maybe someone else helps me out here 🤔

Reply:
btw, you are merging to master and your default branch is main (at least here in remote) 😆.

Reply:
Got it, change commited

Reply:
I'm mergin to master since that is the branch I setted for it, I'm not using main